src/documents/SVGAndAccessibility.md

```diff
@@ -0,0 +1,100 @@
+---
+title: SVG and Accessibility
+subheader:
+layout: default
+category: Methods
+---
+
+## What is SVG?
+
+SVG (Scalable Vector Graphics) is a means of describing vector graphics in markup on a web page. Vector graphics can be advantageous for inclusive design because they can be easily resized, scaled up or down to different resolutions without loss of quality, and tagged with additional semantics that make them compatible with assistive technologies such as screen readers.
```

Comment:
Sentence is a bit long. Consider splitting into two. Perhaps "be easily resized, and scaled up or down to different resolutions without loss of quality. SVGs can also be augmented with additional semantics that will make them compatible with assistive technologies such as screen readers."?